src/messaging/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from users.models import Profile
+# Create your models here.
+
+class Thread(models.Model):
+	profile_1 = models.ForeignKey(Profile, null=True, on_delete=models.SET_NULL, related_name='profile_1')
+	profile_2 = models.ForeignKey(Profile, null=True, on_delete=models.SET_NULL, related_name='profile_2')
+
+class Message(models.Model):
+	thread = models.ForeignKey(Thread, null=True, on_delete=models.SET_NULL)
```

Comment:
i think this should be cascade actually.  in the case of an admin deleting the thread, you'd think all the messages would be deleted, and not just unidentifyable